cypress/pages/ryanairHomepage.cy.js

```diff
@@ -0,0 +1,17 @@
+class ryanairHomepage {
+    elements ={
+        departureButton: () => cy.get('#input-button__departure'),
+        firstDepartureCountry: () => cy.get(':nth-child(2) > :nth-child(1) > .countries__country-inner'),
+        firstDepartureCity: () => cy.get(':nth-child(1) > .body-l-lg > span'),
+        firstDestinationCountry: () => cy.get(':nth-child(3) > :nth-child(2) > .countries__country-inner'),
+        firstDestinationCity: () => cy.get('.list__airports-scrollable-container > .ng-star-inserted > .body-l-lg > span'),
```

Comment:
just poking naming conventions - can you have multiple departure destinations? If somebody without a context would look at it, they'd think you can pick multiple departure spots in order to find out which one of these is more beneficial. Naming is often key, as that's why you get reviews from others because we live in our own biases :)